j2cli/cli.py

```diff
@@ -190,6 +190,7 @@ def render_command(cwd, environ, stdin, argv):
     renderer.register_filters({
         'docker_link': filters.docker_link,
         'env': filters.env,
+        'bool': bool,
```

Comment:
can we have toBool rather than bool?

Reply:
:+1: how about both `bool` and `toBool`? My application of j2cli is using it to format Ansible jinja, and my existing templates all use `bool` :)

Reply:
I would say, bool is a builtin data type so in the future if we add more stuff around the same filter then it will be easier to handle the stuff with a diff name.